lib/utils.js

```diff
@@ -21,4 +21,9 @@ utils.startAtZero = function startAtZero(obj, key) {
   }
 };
 
+utils.isAbsolutePath = require('path').isAbsolute;
+if (!utils.isAbsolutePath) {
+  utils.isAbsolutePath = require('path-is-absolute');
```

Comment:
Do we intend for this to work for windows as well? Windows uses the "\" character for file paths.

https://github.com/sindresorhus/path-is-absolute has a special method for windows.

Also, the example in their README that does 
```
var pathIsAbsolute = require('path-is-absolute');

// Windows
pathIsAbsolute('C:/Users/');
//=> true
```
does not work for me. I have to do
```
pathIsAbsolute.win32('C:/Users/');
```
instead.